cent/exceptions.py

```diff
@@ -0,0 +1,66 @@
+from cent.methods.base import CentMethod, CentType
+
+
+class CentError(Exception):
+    """
+    Wrapper for all exceptions coming from this library.
+    """
+
+
+class CentNetworkError(CentError):
+    """CentNetworkError raised when Centrifugo is not available."""
+
+    def __init__(self, method: CentMethod[CentType], message: str) -> None:
+        self.method = method
+        self.message = message
+
+    def __str__(self) -> str:
+        return f"HTTP error - {self.message}"
+
+    def __repr__(self) -> str:
+        return f"{type(self).__name__}('{self}')"
+
+
+class ClientDecodeError(CentError):
+    """
+    ClientDecodeError raised when response from Centrifugo can't be decoded
+    from JSON.
+    """
+
+
+class InvalidApiKeyError(CentError):
```

Comment:
Let's call it UnauthorizedError (to be more generic for other types of auth, in the message you can write now that API key is invalid)